### PR TITLE
Added Maxhealth and a little cleaning

### PR DESCRIPTION
**Pull request by ThePlatinumGhost**
Added Maxhealth  and you can change the value in the menu
obviously tell me if I did something wrong everything seems to work to me
fix this #4

### CODE REVIEW

File: lua/autorun/sh_vamp_convars.lua [hunk 1]
```diff
@@ -0,0 +1,19 @@
+CreateConVar("ttt2_vamp_bloodtime", "60", {FCVAR_ARCHIVE, FCVAR_NOTIFY})
+CreateConVar("ttt2_vamp_maxhealth", "250", {FCVAR_ARCHIVE, FCVAR_NOTIFY})
+
+hook.Add("TTTUlxDynamicRCVars", "TTTUlxDynamicVampCVars", function(tbl)
```
Comment (TimGoll) [inline lua/autorun/sh_vamp_convars.lua:4]: Please make sure that your identations are constant. They are all over the place in this hook.

File: lua/terrortown/entities/roles/vampire/shared.lua [hunk 2]
```diff
@@ -177,10 +165,13 @@ if SERVER then
...
 
 			local oldHealth = attacker:Health()
-			local heal = math.min(oldHealth + (ply:Health() or 100), math.ceil(oldHealth + dmginfo:GetDamage() * 0.5))
+			if oldHealth < GetConVar("ttt2_vamp_maxhealth"):GetInt() then 
+				local heal = math.min(oldHealth + (ply:Health() or 100), math.ceil(oldHealth + dmginfo:GetDamage() * 0.5))
 
-			attacker:SetMaxHealth(math.max(heal, attacker:GetMaxHealth()))
-			attacker:SetHealth(heal)
+				attacker:SetMaxHealth(math.max(heal, attacker:GetMaxHealth()))
+				if heal > GetConVar("ttt2_vamp_maxhealth"):GetInt() then heal = GetConVar("ttt2_vamp_maxhealth"):GetInt() end
```
Comment (TimGoll) [inline lua/terrortown/entities/roles/vampire/shared.lua]: don't make this oneline please

**Review by TimGoll — CHANGES_REQUESTED** [review]
Thanks for the contribution, a few small minor nitpicks though

File: lua/autorun/sh_vamp_convars.lua (hunk 1 above)
Comment (ThePlatinumGhost) [reply lua/autorun/sh_vamp_convars.lua:4]: They seem constant (i used 1 space) Maybe I did not understand well you request(?)

Comment (TimGoll) [reply lua/autorun/sh_vamp_convars.lua:4]: We use one tab. Therefore your indentation is completely different than the rest of the file

Comment (TimGoll) [reply lua/autorun/sh_vamp_convars.lua:4]: E.g line 7 is a tab, line 8 are spaces

Comment (ThePlatinumGhost) [reply lua/autorun/sh_vamp_convars.lua:4]: Ok so I will make all Tab

Comment (ThePlatinumGhost) [reply lua/autorun/sh_vamp_convars.lua:4]: I changed all the spaces in the tab, see if it is correct now

Comment (TimGoll) [reply lua/autorun/sh_vamp_convars.lua:4]: 11 to 19 are still spaces

Comment (ThePlatinumGhost) [reply lua/autorun/sh_vamp_convars.lua:4]: anyway i think it's all fixed now

**Pull request merged**